tests/test-json-users-controller.php

```diff
@@ -38,6 +38,7 @@ public function test_get_items() {
 		wp_set_current_user( $this->user );
 
 		$request = new WP_JSON_Request( 'GET', '/wp/users' );
+		$request->set_param( 'context', 'view' );
```

Comment:
> Why do you think we should have tests for other contexts?

I think we should have tests for as much things as possible. Do we not need to test both `/wp/users?context=view` and `/wp/users?context=edit`?